### PR TITLE
Anya/71 generate GitHub issue

### PR DESCRIPTION
**Pull request by anyakhvost**
Automatically generate a Github issue when feedback is submitted

Note: we need to create a "support" user in Github and use this user's access_token to create issues.

connects #71

### CODE REVIEW

File: app/services/github_service.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+require "erb"
+
+class GithubService
+  REPO = "department-of-veterans-affairs/appeals-support".freeze
+
+  def create_issue(record)
+    body = GithubIssueRenderer.new(username: record.username,
```
Comment (shanear) [inline app/services/github_service.rb]: maybe it'd be better to move this out to a place where we could test it

then the method would be: 
```
def create_issue(body)
end
```

Then we can do a test that the template is being filled out correctly

File: app/models/feedback.rb [hunk 2]
```diff
@@ -1,9 +1,37 @@
...
+
   validates :subject, :username, :feedback, presence: true
   validates :contact_email, length: { maximum: 255 }, format: { with: /\A\S*@\S*\z/ }, allow_blank: true
   enum status: {
     open: 0,
     in_progress: 1,
     closed: 2
   }
+
+  after_create :create_github_issue
```
Comment (shanear) [inline app/models/feedback.rb]: can we add a feature test for this?

**Review by shanear — CHANGES_REQUESTED** [review]
This is awesome, just 2 comments

File: app/services/github_service.rb (hunk 1 above)
Comment (aroltsch) [inline app/services/github_service.rb]: I am testing the template in github_issue_renderer_spec.rb. However, I think I understand what you mean :). @shanear

File: app/models/feedback.rb (hunk 2 above)
Comment (aroltsch) [inline app/models/feedback.rb]: will do.

**Review by shanear — APPROVED** [review]
👍

**Pull request merged**